bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -1453,7 +1459,9 @@ public void addEntry(ByteBuf entry, WriteCallback cb, Object ctx, byte[] masterK
             bb.flip();
 
             FutureWriteCallback fwc = new FutureWriteCallback();
-            LOG.debug("record fenced state for ledger {} in journal.", ledgerId);
+            if (LOG.isDebugEnabled()) {
```

Comment:
Not a java expert, but isn't there any way to declare a macro to do this job instead of calling it every time??